package.json

```diff
@@ -29,5 +29,8 @@
     "opn": "^5.1.0",
     "prettier": "^1.7.3",
     "rimraf": "^2.6.1"
+  },
+  "engines": {
```

Comment:
This won't really work here as it's the mono repo package json and we should push to at least node 10 (current LTS).